Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ruby:2.7.5-slim
+
+# Install required software:
+#  - net-tools: to run ping and other networking tools
+#  - build-essential: to have a compiling environment for building gems
+#  - curl: for healthcheck
+#  - netcat: for wait for connection to database
+#  - nodejs, yarn, git, default-libmysqlclient-dev and graphviz are rails gems dependencies
+RUN apt-get update && apt-get install -y \
+  net-tools build-essential curl netcat wget vim \
+  nodejs yarn git default-libmysqlclient-dev npm graphviz
```

Comment:
I know it's not what we've always done, but would it be worth splitting these onto separate lines?  I've seen on more than one occasion where duplicates end up in these lists which aren't a problem until you're removing it again in the future and can't understand why removing the one duplicate you spotted doesn't do what you thought it was going to do.  It also gives us the opportunity to put them in A-Z order to help people assess whether the one they want to be there is in fact there, also reducing the chances of duplication.

Reply:
Yes, good idea, I'll do it alphabetical multiline